src/helpers/datatypes.rs

```diff
@@ -0,0 +1,127 @@
+use serde::{Deserialize, Serialize};
```

Comment:
In the verify credential, we created a folder for the datatypes. I think you need to rename this file and move it into the datatypes folder probably.

Reply:
I can't find any datatypes folder in develop branch, can you refer me to branch so that i can move types there ?

Reply:
done, merged the types in the folder.